packages/backend/src/open_payments/client/service.ts

```diff
@@ -45,14 +45,16 @@ export class Receiver extends ConnectionBase {
     return new this(
       incomingPayment.ilpStreamConnection,
       incomingAmount?.value,
-      receivedAmount.value
+      receivedAmount.value,
+      expiryDate
     )
   }
 
   private constructor(
     connection: ConnectionJSON,
     private readonly incomingAmountValue?: bigint,
-    private readonly receivedAmountValue?: bigint
+    private readonly receivedAmountValue?: bigint,
+    private readonly expiresAtValue?: Date
```

Comment:
```suggestion
    public readonly expiresAt?: Date
```
I don't think this needs the getter below

Reply:
Can you update client service / receiver tests to incorporate `expiresAt`?
https://github.com/interledger/rafiki/blob/2c40d222006e5beef2fe850d0208c27c70bfcc69/packages/backend/src/open_payments/client/service.test.ts